File: Problem1/Problem1.py
import time

below = 1000
sum = 0
t01 = time.time()
for x in range(0,below):
    if x % 3 == 0:
        sum = sum + x
    elif x % 5 == 0:
        sum = sum + x
t02 = time.time()
print(sum)

File: Problem1/Problem1.js

var below = 1000
var sum = 0
var x = 0
for ( x = 0; x<below; x++){
    if(x%3==0){
        sum += x;
    }
    else if(x%5==0){
        sum += x;
    }
}
console.log(sum);